st2debug/bin/submit-debug-info

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python2.7
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+
+from st2debug.cmd import submit_debug_info
+
+
+if __name__ == '__main__':
+    sys.exit(submit_debug_info.main())
```

Comment:
The file @tim-ireland is referencing is this one:
* st2debug/bin/st2-submit-debug-info

which we are installing here:
* /opt/stackstorm/virtualenvs/stackstorm/bin/st2-submit-debug-info

The wrapper script asked for in PXPSE-420 should be implemented in the Plexxi/connect repo since it's specific to our implementation of StackStorm. That wrapper script should:
* be named submit-debug-info
* located in packaging/root/usr/sbin/
* invoke /opt/stackstorm/virtualenvs/stackstorm/bin/st2-submit-debug-info
* be on the admin users path
* included in the plexxiconnect package via docker/rpm/SPECS/plexxiconnect.spec